crates/host/src/module.rs

```diff
@@ -104,12 +100,14 @@ pub trait PlruCache {
 
     /// Delete the plru for a given cache key. Care must be taken to ensure this
     /// is not called before a subsequent call to `plru_key` or it will panic.
+    #[allow(dead_code)]
     fn trash(&mut self, key: &CacheKey) {
```

Comment:
Are these really used? It's a private method on a private type now right?

If these are only used in tests I'd prefer `#[cfg(test)]` over `dead_code`

Reply:
No they're not used anywhere, though I thought they might be in the future for a complete cache implementation.

But they are very simple and can be added back if needed. Removed.